<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P3DX Robot GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_ConnectToRobot">
       <property name="text">
        <string>Connect to ROBOT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Rviz2">
       <property name="text">
        <string>Rviz2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Record">
       <property name="text">
        <string>Record  Bag File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_RQt">
       <property name="text">
        <string>RQt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>861</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Autonomous">
       <property name="text">
        <string>Autonomous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ManualControl">
       <property name="text">
        <string>Manual Control</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_NewMap">
       <property name="text">
        <string>New Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_withMap">
       <property name="text">
        <string>Select Map</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>P3DX Robot GUI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DisconnectToRobot">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect to Robot</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>389</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_StartStopRecord">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Strart Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartStopRecord_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveMap">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Map</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>431</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_KeyboardControl">
       <property name="text">
        <string>Keyboard control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_JoyControl">
       <property name="text">
        <string>Joy Stick Contro</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
